core/templates/dev/head/pages/exploration-editor-page/exploration-save-and-publish-buttons/exploration-save-and-publish-buttons.directive.html

```diff
@@ -1,4 +1,4 @@
-<ul class="nav navbar-nav oppia-navbar-nav navbar-right ng-cloak">
+<ul class="nav navbar-nav oppia-navbar-nav navbar-right ng-cloak  hidden-xs hidden-sm">
```

Comment:
Drop extra space before hidden-xs.

Reply:
I have removed the spaces and reordered to alphabetical order.